desktop/cmp/tab/pane/TabModel.js

```diff
@@ -64,6 +73,18 @@ export class TabModel {
         this.lastRefreshRequest = Date.now();
     }
 
+    @action
+    setDisabled(disabled) {
+        this.disabled = disabled;
+        if (disabled && this.isActive) {
+            const {containerModel} = this,
+                tab = containerModel.firstEnabledTab;
+
+            throwIf(!tab, 'Could not find an enabled tab to activate!');
```

Comment:
Ideally, you might throw *before* disabling tab if you see that all tabs will be disabled.    Its a real corner case, but in general we want to throw *before* we actually change the state, if the change is going to be untenable.